projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-posts-v1-1-endpoint.php

```diff
@@ -220,6 +221,10 @@ public function callback( $path = '', $blog_id = 0 ) {
 			$query['has_password'] = false;
 		}
 
+		if ( isset( $args['include'] ) ) {
+			$query['post__in'] = $args['include'];
```

Comment:
Should we worry about other parameters that may already set `post__in`, such as https://github.com/Automattic/jetpack/blob/1a7961cde8fa002e29395fa181123801fff4b19a/projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-posts-v1-1-endpoint.php#L251-L254

Maybe we should merge the 2, like we do with `post__not_in` elsewhere in this file?

Reply:
Good catch! [Fixed it here](https://github.com/Automattic/jetpack/pull/29149/commits/22278727539e13044e1ae3dc7cdc327fa1906060).